dev/reference/sql/functions-and-operators/window-functions.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 窗口函数
+category: reference
+aliases: ['/docs-cn/sql/window-functions/']
+---
+
+# 窗口函数
+
+TiDB 中窗口函数的使用方法与 MySQL 基本一致，详情可参见 [MySQL 窗口函数](https://dev.mysql.com/doc/refman/8.0/en/window-functions.html)。使用窗口函数时需将 `tidb_enable_window_function` 设置为 `1`，其默认值为 `0`。
```

Comment:
We are going to make window function general available in releaase 3.0. This variable should be set to 1 by default.